service/prometheus/prometheus.go

```diff
@@ -10,10 +10,6 @@ const (
 	// that the prometheus-config-controller should scrape.
 	ClusterAnnotation = "giantswarm.io/prometheus-cluster"
 
-	// ClusterLabel is the Prometheus label used to identify jobs
-	// managed by the prometheus-config-controller.
-	ClusterLabel = "prometheus_config_controller"
```

Comment:
We originally had both `prometheus_config_controller` and `cluster_id` labels, with the `prometheus_config_controller` label identifying a job as being controlled by the `prometheus-config-controller`. However, we changed this to use just `cluster_id`, as both labels weren't needed, and didn't get round to cleanup because of code freeze.